compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -116,9 +116,9 @@ object Parsers {
     /** Issue an error at given offset if beyond last error offset
       *  and update lastErrorOffset.
       */
-    def syntaxError(msg: => Message, offset: Int = in.offset): Unit =
+    def syntaxError(msg: => Message, offset: Int = in.offset, name: Name = in.name): Unit =
```

Comment:
Restored it, where `length = the length of implicit keyword`, otherwise it is the length of the identifier (in this case 3, from `Int`(.